README.md

```diff
@@ -1,34 +1,41 @@
-# VC Test Suite Implementations _(vc-test-suite-implementations)_
+<!--
+Copyright 2024 Digital Bazaar, Inc.
```

Comment:
It's 2025 and all dates in these changes show 2024.

Reply:
📋